src/migration/dev-1.js

```diff
@@ -48,7 +48,7 @@ export default {
     {
       name: "Beekeeper's Database",
       connectionType: "sqlite",
-      defaultDatabase: path.join(platformInfo.userDirectory, 'app.db')
+      defaultDatabase: path.join(platformInfo.userDirectory, platformInfo.isDevEnv ? 'app-dev.db' : 'app.db')
```

Comment:
Please just have this look at platformInfo.appDbPath.

We should def have a single place we define this.